ExampleApp iOS/flowforms/Extensions/FlowForm+extension.swift

```diff
@@ -0,0 +1,84 @@
+//
+//  FlowForm+extension.swift
+//  FlowForms
+//
+//  Created by Tarek Radovan on 11/07/2023.
+//  Copyright © 2023 orgName. All rights reserved.
+//
+
+import Foundation
+import shared
+import SwiftUI
+import KMPNativeCoroutinesCombine
+import KMPNativeCoroutinesAsync
+
+// MARK: FFCFlowForm
+/// This extension provides additional functionality for the FlowForm class.
+extension FlowForm {
+  
+  /// Binds the status of the form with a publisher.
+  /// - Parameters:
+  ///   - publisher: The publisher to bind the status with.
+  func bindStatus(withPublisher publisher: inout Published<String>.Publisher) {
+    let formPublisher = createPublisher(for: status)
+    
+    formPublisher
+      .receive(on: DispatchQueue.main)
+      .map({ status in
+        return status.code
+      })
+      .replaceError(with: StatusCodes.shared.INCORRECT)
+      .assign(to: &publisher)
+  }
+  
+  /// Binds a field with the validation of the respective ID for that field.
+  /// - Parameters:
+  ///   - field: The field to bind.
```

Comment:
Since this variable was changed to `fieldValue`, can we update this doc to say something like this?
`- fieldValue: The current field's value.`